core/utils/metadata-utils/src/test/java/datawave/query/util/MetadataHelperTest.java

```diff
@@ -104,6 +104,13 @@ private void givenMutation(String row, String columnFamily, String columnQualifi
         givenMutation(mutation);
     }
 
+    private void givenHiddenField(String row, String datatype) {
+        Text h = new Text("h");
```

Comment:
Nit: You could still use `ColumnFamilyConstants.COLF_H` here but I approved. Maybe if you make changes for someone else throw that in?